src/robot/result/model.py

```diff
@@ -264,6 +264,29 @@ def doc(self):
         return ''
 
 
+@Body.register
+class Continue(model.Continue, StatusMixin, DeprecatedAttributesMixin):
+    __slots__ = ['status', 'starttime', 'endtime']
+
+    def __init__(self, values=(), status='FAIL', starttime=None, endtime=None, parent=None):
+        super().__init__(values, parent)
```

Comment:
`Continue` shouldn't accept `values`. Same also with `Break`.

Reply:
I had checked this, and I was thinking that it will be a empty, so did not remove. 
Removing now.